src/components/heroImage/heroImage.scss

```diff
@@ -12,15 +12,17 @@ nav {
 }
 #hero-image {
     height: 80vh;
-    width: 100vw;
+    width: 100%;
 }
 #hero-circle {
     position: absolute;
     border-radius: 50%;
-    background: rgba(179,175,175,.95);
-    opacity: .25;
-    width: 60vh;
-    height: 60vh;
+    background: rgba(179, 175, 175, 0.95);
+    opacity: 0.25;
+    width: 60vw;
```

Comment:
I see what you are doing here, and I like it. However we want to try to maintain the size of the circle as log as possible so it doesn't shrink too quickly with skinnier windows. This should probably be 80vw or 90vw. same with the below height: 60vw. Should also probably be 80 or 90.

Reply:
Just played around with it a bit in Chrome inspector. 80vw is the sweetspot